eng/scripts/InstallJdk.ps1

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
@riarenas @MattGal it seems like something wrong is happening on our agents w.r.t. to installing the JDK. This script is just downloading a zip and extracting it, but is currently taking as long as 13 minutes. The file being downloaded is only 170 MB, so thats ~250 kBps download speed. My phone can download this faster. Is there something wrong with our VM config?

https://netcorenativeassets.blob.core.windows.net/resource-packages/external/windows/java/jdk-11.0.1_windows-x64_bin.zip

Reply:
![image](https://user-images.githubusercontent.com/2696087/51005917-b18f1c80-14f5-11e9-9bb0-f25fd9c45ee2.png)

Reply:
I can confirm that both the VM and storage account are in the West US data center.  Could you add some logging to the inline ps1 used?  I'm thinking there may be more than just the download making this take 9-13 mins.

Reply:
Done. Added a log line between each step. Let's watch https://dnceng.visualstudio.com/public/_build/results?buildId=70656 and see what happens.

Reply:
I think it's actually just being slow... watching the graphs on the storage account itself there's a huge spike in egress and latency when you kick off your builds.

Reply:
ah ha.   Please change
```
Invoke-WebRequest -UseBasicParsing -Uri "https://netcorenativeassets.blob.core.windows.net/resource-packages/external/windows/java/jdk-${JdkVersion}_windows-x64_bin.zip" -Out "$tempDir/jdk.zip"
```

to 
```
$ProgressPreference = 'SilentlyContinue'
Invoke-WebRequest -UseBasicParsing -Uri "https://netcorenativeassets.blob.core.windows.net/resource-packages/external/windows/java/jdk-${JdkVersion}_windows-x64_bin.zip" -Out "$tempDir/jdk.zip"
```
and perf should get about 100x better.  See https://github.com/PowerShell/PowerShell/issues/2138  for a discussion.

Reply:
Holy smokes, that's a crazy bug. From 10-12 minutes to 22 seconds. Excellent find @MattGal !